Grooming/Grooming.md

```diff
@@ -2,42 +2,48 @@
 ```bash
 {
 "**Prompt:**" "How can I schedule a grooming appointment for my pet through Petofy?"
+"**Response:**": "To schedule a grooming session visit https://www.petofy.com/. Now follow these steps:-
+ - Go to Grooming under Find services
+ - Select your city from the list.
+ - Select the groomer who meets your criteria
+Make sure are logged into your Petofy account for booking the groomer."
+ 
 "**Response:**" "You can simply Log in to your Petofy account, add your location, go to the grooming section, choose your preferred groomer, and select a convenient appointment slot."
```

Comment:
@ANMOL2001A As you are making a "visitor" to reach to a service, you must make this stepwise